app/Services/Model/Imp/SummitOrderService.php

```diff
@@ -4338,4 +4349,40 @@ public function deActivateTicket(Summit $summit, int $order_id, int $ticket_id):
             return $ticket;
         });
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function renderOrderConfirmationEmail(Summit $summit, int $order_id): string
+    {
+        try {
+            Log::debug("SummitOrderService::renderOrderConfirmationEmail order id {$order_id}");
+
+            $order = $summit->getOrderById($order_id);
+
+            if (!$order instanceof SummitOrder)
+                throw new EntityNotFoundException("order not found");
```

Comment:
errors should start with capital letter and end with .

Reply:
done